src/service/mobile_config.proto

```diff
@@ -83,6 +83,49 @@ message gateway_info_v2 {
   uint64 updated_at = 5;
 }
 
+// Warning: Be careful, values in v2 differ from v1
+enum device_type_v2 {
+  indoor = 0;
+  outdoor = 1;
+  data_only = 2;
+}
+
+message location_info {
+  // The res12 h3 index asserted address of the gateway as a string
+  string location = 1;
+  // The Unix epoch timestamp (in seconds) when the location was last changed
+  uint64 location_changed_at = 2;
+}
+
+message deployment_info {
+  // The antenna ID of the gateway
+  uint32 antenna = 1;
+  // The height of the hotspot above ground level, in whole meters
+  uint32 elevation = 2;
+  // The azimuth (direction) of the antenna, in degrees
+  uint32 azimuth = 3;
```

Comment:
Are we dealing with direction in whole units? 
Does this need to be a float, or are we treating it to some sort of decimal place?

Reply:
> Are we dealing with direction in whole units?

Yes

> Does this need to be a float

No, it doesn't. This field is always a whole number in the `mobile_hotspot_infos` table